Adyen.xcodeproj/project.pbxproj

```diff
@@ -5965,9 +5954,9 @@
 				);
 				MARKETING_VERSION = 4.6.1;
 				OTHER_SWIFT_FLAGS = "-Xfrontend -warn-long-expression-type-checking=100";
-				PRODUCT_BUNDLE_IDENTIFIER = com.adyen.AdyenUIHost;
+				PRODUCT_BUNDLE_IDENTIFIER = com.adyen.CheckoutDemoUIKit;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "HostUI Testing with ApplePay";
+				PROVISIONING_PROFILE_SPECIFIER = "Checkout Demo UIKit Distribution";
```

Comment:
![Screenshot 2022-01-13 at 13 03 26](https://user-images.githubusercontent.com/2648655/149327697-649861b5-b217-4c2b-818c-3c3b98e2913c.png)

Reply:
Maybe we should use another / refreshed certificate ?

Reply:
Resolved. Talked offline